asreview/webapp/src/ProjectComponents/ExportComponents/ExportPage.js

```diff
@@ -314,6 +330,59 @@ const ExportPage = (props) => {
                   </span>
                 </Tooltip>
               </Box>
+              .
+                <StyledActionsBox>
+                  <Box borderRadius={3} padding={2} maxWidth="600px">
```

Comment:
Do we need the width hardcoded?

Reply:
maxWidth: { xs: '80%', sm: '40%', md: '30%' }, width: '100%'